azkaban-common/src/main/java/azkaban/executor/container/ContainerImplUtils.java

```diff
@@ -60,7 +61,9 @@ private static void populateJobTypeForFlow(final ExecutableNode node, Set<String
         populateJobTypeForFlow(subNode, jobTypes);
       }
     } else {
-      jobTypes.add(node.getType());
+      if (node.getStatus() != Status.DISABLED) {
```

Comment:
Can you please add comment? Also log for which is disabled and not added? It might help later on for debugging.

Reply:
Good point! Done.